package.json

```diff
@@ -115,6 +122,7 @@
     "sass-loader": "6.0.6",
     "scratch-gui": "0.1.0-prerelease.20190711011201",
     "scratch-l10n": "latest",
+    "selenium-webdriver": "3.6.0",
```

Comment:
I know many other dependencies are set to the "bug" level of change (X.X.X), but aren't we basically ok with not stressing about those tiny version changes? Can we do (~3.6.0) instead, since we don't expect breaking changes below the major level (and tilde would be even more careful, since it would still bug us about minor changes)

Reply:
In this case I just moved the selenium-webdriver package from the integration folder where it was in a separate package.json.  I didn't modify it at all in the move.  For what it's worth, the last time this package changed was two years ago.   A newer version has been in alpha for about that long and will likely come out as 4.0.0.

We could change it, but I'm not sure how much it'll matter.

Reply:
OK